src/libraries/System.Drawing.Primitives/tests/ColorTests.cs

```diff
@@ -889,5 +889,27 @@ private static int GetColorRefValue(Color color)
             // The COLORREF value has the following hexadecimal form: 0x00bbggrr.
             return color.B << 16 | color.G << 8 | color.R;
         }
+
+        [Fact]
+        public void SystemColor_AlternativeColors()
```

Comment:
Please add test for high-contrast.

Reply:
I was imagining that you could make some test hook and do it with remote-executor?

Reply:
It would still change the state for all processes, so I'm not sure that would buy much.

Reply:
I was imagining you had some hook with state that you could set in a static, but I guess at that point you're just testing your hook rather than the actual API.  If you had to cache the HC value then it might be less of a fake, but I see from https://github.com/dotnet/runtime/pull/105525#discussion_r1695722873 that you mention you can't cache it.  Your call here just make sure you validate the HC behavior somewhere.